components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa/src/Publisher/Models/Events/FilterSimpleAttributeModel.cs

```diff
@@ -0,0 +1,34 @@
+﻿namespace Microsoft.Azure.IIoT.OpcUa.Publisher.Config.Models.Events {
+    using System.Collections.Generic;
+    using System.Runtime.Serialization;
+
+    /// <summary>
+    /// Class to describe the SimpleAttributeOperand.
```

Comment:
let the comment reflect the class name

Reply:
Comment is actually accurate. The class FilterSimpleAttributeModel describes SimpleAttributeOperand in the spec. I'm more inclined to change the class name according to spec. We have more of these issues however that I have been holding off on. Thoughts?

Reply:
Why can't we use SimpleAttributeOperandModel here instead? This is a duplicate.

Reply:
fix: reused existing model classes